{"timestamp":"2025-08-19T18:34:32.168871","level":"info","event":"DAG bundles loaded: dags-folder","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-08-19T18:34:32.171689","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/test-minio.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-08-19T18:34:32.262207","level":"info","event":"Secrets backends loaded for worker","count":1,"backend_classes":["EnvironmentVariablesBackend"],"logger":"supervisor"}
{"timestamp":"2025-08-19T18:34:32.297262","level":"info","event":"Connection Retrieved 'minio_s3'","logger":"airflow.hooks.base"}
{"timestamp":"2025-08-19T18:34:32.297672","level":"info","event":"AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.","logger":"airflow.providers.amazon.aws.utils.connection_wrapper.AwsConnectionWrapper"}
{"timestamp":"2025-08-19T18:34:45.539722","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"EndpointConnectionError","exc_value":"Could not connect to the endpoint URL: \"http://localhost:9001/test-minio/test.txt\"","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":825,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1088,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":408,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/decorator.py","lineno":251,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":408,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":212,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":235,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/test-minio.py","lineno":12,"name":"upload"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/amazon/aws/hooks/s3.py","lineno":153,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/amazon/aws/hooks/s3.py","lineno":126,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/amazon/aws/hooks/s3.py","lineno":1192,"name":"load_string"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/amazon/aws/hooks/s3.py","lineno":1275,"name":"_upload_file_obj"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/boto3/s3/inject.py","lineno":642,"name":"upload_fileobj"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/s3transfer/futures.py","lineno":103,"name":"result"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/s3transfer/futures.py","lineno":264,"name":"result"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/s3transfer/tasks.py","lineno":135,"name":"__call__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/s3transfer/tasks.py","lineno":158,"name":"_execute_main"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/s3transfer/upload.py","lineno":796,"name":"_main"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/client.py","lineno":569,"name":"_api_call"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/client.py","lineno":1005,"name":"_make_api_call"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/client.py","lineno":1029,"name":"_make_request"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/endpoint.py","lineno":119,"name":"make_request"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/endpoint.py","lineno":200,"name":"_send_request"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/endpoint.py","lineno":360,"name":"_needs_retry"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/hooks.py","lineno":412,"name":"emit"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/hooks.py","lineno":256,"name":"emit"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/hooks.py","lineno":239,"name":"_emit"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/retryhandler.py","lineno":207,"name":"__call__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/retryhandler.py","lineno":284,"name":"__call__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/retryhandler.py","lineno":320,"name":"_should_retry"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/retryhandler.py","lineno":363,"name":"__call__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/retryhandler.py","lineno":247,"name":"__call__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/retryhandler.py","lineno":416,"name":"_check_caught_exception"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/endpoint.py","lineno":279,"name":"_do_get_response"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/endpoint.py","lineno":383,"name":"_send"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/httpsession.py","lineno":493,"name":"send"}]},{"exc_type":"NewConnectionError","exc_value":"<botocore.awsrequest.AWSHTTPConnection object at 0x7fd7ac96f560>: Failed to establish a new connection: [Errno 111] Connection refused","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/httpsession.py","lineno":464,"name":"send"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/connectionpool.py","lineno":841,"name":"urlopen"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/util/retry.py","lineno":449,"name":"increment"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/util/util.py","lineno":39,"name":"reraise"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/connectionpool.py","lineno":787,"name":"urlopen"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/connectionpool.py","lineno":493,"name":"_make_request"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/awsrequest.py","lineno":96,"name":"request"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/connection.py","lineno":445,"name":"request"},{"filename":"/usr/local/lib/python3.12/http/client.py","lineno":1333,"name":"endheaders"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/awsrequest.py","lineno":123,"name":"_send_output"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/awsrequest.py","lineno":223,"name":"send"},{"filename":"/usr/local/lib/python3.12/http/client.py","lineno":1037,"name":"send"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/connection.py","lineno":276,"name":"connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/connection.py","lineno":213,"name":"_new_conn"}]},{"exc_type":"ConnectionRefusedError","exc_value":"[Errno 111] Connection refused","exc_notes":[],"syntax_error":null,"is_cause":true,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/connection.py","lineno":198,"name":"_new_conn"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/util/connection.py","lineno":85,"name":"create_connection"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/urllib3/util/connection.py","lineno":73,"name":"create_connection"}]}]}
